README.md

```diff
@@ -1,65 +1,133 @@
+If the manual compilation was successful, you should see a JAR file in the `target` directory, with a name like 
+`data-exporter-VERSION_NUMBER-jar-with-dependencies.jar`. This is the file you will run with a command like the 
+following (the `-c|--overwrite_config_file flag is optional and forces a new configuration file to be created through
+prompts from the user for new values).
 
-## Configuration
+2. `java -jar target/data-exporter-1.0-SNAPSHOT-jar-with-dependencies.jar [-c|--overwrite_config_file]`
 
-**NOTE: Configuration is done when calling the main script `runDataExporter.sh` for the first time, so this section is if set values need to be changed and for general reference.**
+## Configuration
 
-A auto-configuration script is provided at the root directory of this project and is called `configureDataExporter.sh`.  On first running the Icon Finder tool the configuration script will be run automatically and create a configuration file.   If the configuration needs to be changed, however, this script can be run directly: `./configureDataExporter.sh`.
+**NOTE: Configuration is done when calling the main script `runDataExporter.sh` for the first time (via invocation of
+the `data-exporter.jar`), so this section is if set values need to be changed as well as for general reference.**
 
-The configuration file produced by the script will be at the root directory of this project and named `config.properties`.  It can be viewed and edited directly if desired.
+The configuration file produced will be at the root directory of this project and named `config.properties`.  It can 
+be viewed and edited directly if desired.
 
-A sample configuration file is provided at `src/main/resources/config.properties` and looks like this, but should **NEVER BE EDITED DIRECTLY** and any changes are ignored by git after running the configuration script.
+A sample configuration file is provided at `src/main/resources/config.properties` and looks like this, but should 
+**NEVER BE EDITED DIRECTLY** and any changes are ignored by git after running the configuration script.
 
 ```
-user=neo4j_user
-pass=neo4j_pass
-host=localhost
-port=7687 # Default Bolt port for Neo4J
-reactomeVersion=67
-outputDir=data-exporter/output
+neo4jUserName=neo4j
+neo4jPassword=root
+neo4jHostName=localhost
+neo4jPort=7687
+
+releaseNumber=74
+outputDir=output
+
+ncbiFTPUserName=ncbi_ftp_reactome_account_user_name
+ncbiFTPPassword=ncbi_ftp_reactome_account_password
+ncbiFTPHostName=ncbi_ftp_server_host_name
+ncbiFTPDirectory=path/to/reactome/files/on/ncbi/ftp/server
+
+europePMCFTPUserName=europe_pmc_reactome_account_user_name
+europePMCFTPPassword=europe_pmc_reactome_account_password
+europePMCFTPHostName=europe_pmc_ftp_server_host_name
+europePMCFTPDirectory=path/to/reactome/files/on/europe_pmc/ftp/server
 ```
 
+## Integration Tests
+
+The source code for this program has integration tests which require connection to the NCBI and EuropePMC FTP Servers 
```

Comment:
Do the integration tests require login information for these servers? If so, where will future developers find those credentials? Not in github, I assume, but where?

Reply:
The login information is a kind of Reactome internal knowledge and doesn't have a specific place other than in the configuration file of the server's copy of the repository (similar to AddLinks and other repos that need credentials but which can't be stored in version control).  

It also exists in the Reactome Perl module that holds sensitive configuration information locally and this information can be obtained from Reactome team members.